src/main/java/org/jskele/libs/dao/impl/mappers/JsonValueConverter.java

```diff
@@ -1,20 +0,0 @@
-package org.jskele.libs.dao.impl.mappers;
-
-import com.google.common.base.Preconditions;
-import org.jskele.libs.dao.JsonValue;
-import org.postgresql.util.PGobject;
```

Comment:
now postgres insn't needed for compiling any more (should remove dependency as well)

Reply:
fixed